internal/services/v0/acl.go

```diff
@@ -258,20 +239,13 @@ func (as *aclServer) Read(ctx context.Context, req *v0.ReadRequest) (*v0.ReadRes
 }
 
 func (as *aclServer) Check(ctx context.Context, req *v0.CheckRequest) (*v0.CheckResponse, error) {
-	atRevision, err := as.pickBestRevision(ctx, req.AtRevision)
```

Comment:
It's a little unclear why `Check` and `ContentChangeCheck` will behave differently at first glance. I wonder if we could improve the ergonomics at the call site of `MustRevisionFromContext`.

Reply:
Is it worth doing much more than a comment here, since it's the v0 api? (I think the consistency block logic is much clearer in v1).

Reply:
(added a comment to ContentChangeCheck)